src/main/java/com/nhantran/pages/TimetablePage.java

```diff
@@ -1,13 +1,21 @@
 package com.nhantran.pages;
 
+import com.nhantran.enums.RailwayStations;
 import com.nhantran.utils.SeleniumActions;
 import org.openqa.selenium.By;
 
 public class TimetablePage extends BasePage {
 
-    private String checkPriceLink = "//tr[td[text()='%s' and following-sibling::td[text()='%s']]]//a[text()='check price']";
+    private String lnkCheckPrice = "//tr[td[text()='%s' and following-sibling::td[text()='%s']]]//a[text()='check price']";
+    private String linkBookTicket = "//tr[td[text()='%s' and following-sibling::td[text()='%s']]]//a[text()='book ticket']";
 
-    public void clickCheckPriceLink(String departStation, String arrivalStation) {
-        SeleniumActions.clickElement((By.xpath(String.format(checkPriceLink, departStation, arrivalStation))));
+    public void clickCheckPriceLink(RailwayStations departStation, RailwayStations arrivalStation) {
+        SeleniumActions.scrollToElement((By.xpath(String.format(lnkCheckPrice, departStation.getValue(), arrivalStation.getValue()))));
```

Comment:
create a variable to assign for this: String.format(lnkCheckPrice, departStation.getValue(), arrivalStation.getValue()

=> reduce duplicate